src/panels/config/zha/zha-clusters.ts

```diff
@@ -0,0 +1,168 @@
+import { fireEvent } from "../../../common/dom/fire_event";
+
+import {
+  Cluster,
+  ZHADeviceEntity,
+  fetchClustersForZhaNode,
+} from "../../../data/zha";
+
+declare global {
+  // for fire event
+  interface HASSDomEvents {
+    "zha-cluster-selected": {
+      cluster?: Cluster;
+    };
+  }
+}
+
+export class ZHAClusters extends LitElement {
+  public hass?: HomeAssistant;
+  public isWide?: boolean;
+  public showHelp: boolean;
+  public selectedEntity?: ZHADeviceEntity;
+  private _selectedClusterIndex: number;
+  private _clusters: Cluster[];
+  private _haStyle?: DocumentFragment;
+  private _ironFlex?: DocumentFragment;
+
+  constructor() {
+    super();
+    this.showHelp = false;
+    this._selectedClusterIndex = -1;
+    this._clusters = [];
+  }
+
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      isWide: {},
+      showHelp: {},
+      selectedEntity: {},
+      _selectedClusterIndex: {},
+      _clusters: {},
+    };
+  }
+
+  protected update(changedProperties: PropertyValues): void {
+    if (changedProperties.has("selectedEntity")) {
+      this._clusters = [];
+      this._selectedClusterIndex = -1;
+      fireEvent(this, "zha-cluster-selected", {
+        cluster: undefined,
+      });
+      this._fetchClustersForZhaNode();
+    }
+    super.update(changedProperties);
+  }
+
+  protected render(): TemplateResult {
+    return html`
+      ${this._renderStyle()} ${this._renderClusterPicker()}
+    `;
+  }
+
+  private _renderClusterPicker(): TemplateResult {
+    return html`
+      <div class="node-picker">
+        <paper-dropdown-menu dynamic-align="" label="Clusters" class="flex">
+          <paper-listbox
+            slot="dropdown-content"
+            .selected="${this._selectedClusterIndex}"
+            @iron-select="${this._selectedClusterChanged}"
+          >
+            ${
+              this._clusters.map(
+                (entry) => html`
+                  <paper-item>${this._computeClusterKey(entry)}</paper-item>
+                `
+              )
+            }
+          </paper-listbox>
+        </paper-dropdown-menu>
+      </div>
+      <div ?hidden="${!this.showHelp}" class="helpText">
+        Select cluster to view attributes and commands
+      </div>
+    `;
+  }
+
+  private async _fetchClustersForZhaNode(): Promise<void> {
+    if (this.hass) {
+      this._clusters = await fetchClustersForZhaNode(
+        this.hass,
+        this.selectedEntity!.entity_id,
+        this.selectedEntity!.device_info!.identifiers[0][1]
+      );
+    }
+  }
+
+  private _selectedClusterChanged(event: ItemSelectedEvent): void {
+    this._selectedClusterIndex = event.target!.selected;
+    fireEvent(this, "zha-cluster-selected", {
+      cluster: this._clusters[this._selectedClusterIndex],
+    });
+  }
+
+  private _computeClusterKey(cluster: Cluster): string {
+    return (
+      cluster.name + " (id: " + cluster.id + ", type: " + cluster.type + ")"
```

Comment:
Prefer backticks

```suggestion
return `${cluster.name} (id: ${cluster.id}, type: ${cluster.type})`;
```

Since this function doesn't use `this`, place it above the class.